locales/en-US/strings.properties

```diff
@@ -115,6 +115,19 @@ time_label_day={number}d
 # in English, while "Home" should be localized matching the about:preferences
 # sidebar mozilla-central string for the panel that has preferences related to
 # what is shown for the homepage, new windows, and new tabs.
+prefs_new_windows_tabs_header=New Windows and Tabs
+prefs_new_windows_description=Choose what you see when you open your homepage, new windows, and new tabs.
+
+prefs_homepage_choice_label=Homepage and new windows:
+prefs_newtab_choice_label=New tabs:
+prefs_homepage_newtab_choice_1_firefox_home=Firefox Home
```

Comment:
The design has a grayed out " (Default)" after this "Firefox Home" text. Last I chatted with design, it sounded like it's okay with it just being regular text. Not sure if you were planning on some other way to add that text? Or is it not desired anymore?